apps/communications/dialer/test/marionette/lib/views/keypad/views.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+/* global module */
+var KeypadAccessor = require('./accessors');
+
+function KeypadView(client) {
+  
+  this.client = client;
+  this.accessors = new KeypadAccessor(client);
+
+}
+
+KeypadView.prototype = {
+  
+  typePhoneNumber: function(number) {
+    var i;
+    var typeNumberArray = String(number).split('');
+
+    for (i in typeNumberArray){
+      if(i == '+'){
```

Comment:
nit: Leave a space between the ``if`` and the opening parentheses and between the closing parentheses and the opening brace. Note that use of ``===`` is preferred over ``==`` in gaia unless you explicitly need the behavior of ``==``